.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1262,42 +1262,57 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             var ret = new DateTimeResolutionResult();
 
             var er = this.config.DateExtractor.Extract(text, referenceDate);
+            var pr1 = new DateTimeParseResult();

```

Comment:
Why do you need to create two empty objects? Isn't it enough to declare the variable type?